Gemfile

```diff
@@ -15,3 +15,6 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
@bf4 it's just copy from [Railties app template](https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/Gemfile#L56), not known jruby status.